example/lib/line_chart/samples/line_chart_sample1.dart

```diff
@@ -396,8 +396,14 @@ class LineChartSample1State extends State<LineChartSample1> {
         barWidth: 2,
         isStrokeCapRound: true,
         dotData: FlDotData(
-          show: true,
-        ),
+            show: true,
+            getDotDrawer: (spot, percentage, data, index) {
```

Comment:
And about here, our feature shouldn't break anything, I mean it should work as same before (because we've added just a new type, and previous one is still supported).
I mean just by setting `FlDotData.show = true`, it should work like the previous version.